src/components/Worker.tsx

```diff
@@ -0,0 +1,111 @@
+                                vel vehicula nisl. Ut mauris metus, fermentum nec orci nec,
+                                sagittis laoreet eros. Nam porta luctus mi, sodales aliquam nibh
+                                convallis ac. Nunc tempor malesuada lobortis. Pellentesque
+                                habitant morbi tristique senectus et netus et malesuada fames ac
+                                turpis egestas. Phasellus eu tristique enim. Sed id malesuada
+                                velit. Suspendisse ut nisi quis neque venenatis vestibulum id non
+                                ipsum. Aliquam aliquet dapibus nisi, ut tempor massa hendrerit ut.
+                                Sed efficitur ex posuere, interdum turpis a, maximus urna. Aenean
+                                sed sapien nec erat viverra lobortis cursus sit amet ex. Maecenas
+                                vel tempus mi, volutpat maximus tortor. Suspendisse tempus eget
+                                ligula id mattis. Nam porttitor mauris a accumsan convallis.
+                            </div>
+                            <br />
+                            <ul className="list-inline social-buttons">
+                                <li>
+                                    <a href="https://facebook.com" target="_blank">
+                                        <img
+                                            src="../../img/img/facebook.png"
+                                            className="icon"
+                                            height="40px"
+                                            width="40px"
+                                        />
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="https://ca.linkedin.com" target="_blank">
+                                        <img 
+                                            src="../../img/img/linkedin.png" 
+                                            height="40px"
+                                            width="40px"
+                                        />
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="https://twitter.com" target="_blank">
+                                        <img
+                                            src="../../img/img/twitter-x-logo.png"
+                                            className="icon"
+                                            height="40px"
+                                            width="40px"
+                                        />
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </>
+    )
+}
+
+export default Worker;
```

Comment:
We do not use export default but only regular export. Change it here to the regular export and find me a reason why it is better than using export default.